onnxruntime/core/providers/cuda/generator/range.h

```diff
@@ -3,8 +3,7 @@
 
 #pragma once
 
-#include "core/common/common.h"
-#include "core/framework/op_kernel.h"
+#include "core/providers/cuda/cuda_kernel.h"
```

Comment:
>oviders/cuda/cuda_kernel.h" [](start = 17, length = 27)

We could get by with forward declarations

Reply:
we could. however, it says to avoid them here: [https://google.github.io/styleguide/cppguide.html#Forward_Declarations](https://google.github.io/styleguide/cppguide.html#Forward_Declarations)

---
In reply to: [542671088](https://github.com/microsoft/onnxruntime/pull/6118#discussion_r542671088) [](ancestors = 542671088)